src/components/Form/Form.js

```diff
@@ -0,0 +1,54 @@
+import "./Form.style.css";
+import Button from "../Button/Button.js";
+
+export default function Form({ onAddActivity }) {
+	function handleSubmit(e) {
+		e.preventDefault();
+		const formData = new FormData(e.target);
```

Comment:
We do not want to use abbreviations cause code is easier to read and understand without them 🙂 

```suggestion
	function handleSubmit(event) {
		event.preventDefault();
		const formData = new FormData(event.target);
```

Reply:
Changed it